src/main/scala/uk/gov/homeoffice/drt/ResourceObject.scala

```diff
@@ -0,0 +1,50 @@
+package uk.gov.homeoffice.drt
+
+import cats.effect.{Concurrent, ContextShift, Resource, Sync}
+import cats.syntax.all._
+import natchez.Trace.Implicits.noop
+import org.slf4j.LoggerFactory
+import skunk.Session
+import uk.gov.homeoffice.drt.coders.AirlineDecoder
+import uk.gov.homeoffice.drt.model.Airlines
+import uk.gov.homeoffice.drt.service.AirlineService
+
+import scala.io.Source
+
+object ResourceObject {
```

Comment:
Could we think up a better name for this? It's a bit generic as it is

Reply:
May be ApplicationResource as it is get resources we need in application